core/src/main/kotlin/org/evomaster/core/search/gene/SqlForeignKeyGene.kt

```diff
@@ -142,4 +142,22 @@ class SqlForeignKeyGene(
         this.uniqueIdOfPrimaryKey = other.uniqueIdOfPrimaryKey
     }
 
+    /**
+     * Reports if the current value of the fk gene is NULL.
+     * This procedure should be called only if the pk is valid.
+     */
+    fun isNull(): Boolean {
+        if (!hasValidUniqueIdOfPrimaryKey()) {
+            throw IllegalStateException("Cannot ")
```

Comment:
"Cannot " does not give some much info :) could write little more

Reply:
Sorry. I guess the rest somehow got deleted.